recipes-core/ncurses/ncurses_%.bbappend

```diff
@@ -0,0 +1 @@
+CACHED_CONFIGUREVARS_append_toolchain-clang = " cf_cv_prog_CC_c_o=yes cf_cv_prog_CXX_c_o=yes"
```

Comment:
it is on hard-float arm. Anywhere else I've tested it works fine without the ncurses fix.